src/Menu/PanelMenu.tsx

```diff
@@ -49,8 +50,9 @@ export class PanelMenu extends SceneObjectBase<PanelMenuState> implements VizPan
           // removing legendFormat to get verbose legend in Explore
           delete query.legendFormat;
         });
-        const trail = getTrailFor(this);
-        exploreUrl = getExploreURL(panelData, trail, panelData.timeRange);
+        // use the metric scene to get the explore url with interpolated variables
+        const metricScene = sceneGraph.getAncestor(this, MetricScene);
```

Comment:
This works in the Metric scene, but unfortunately it doesn't work in the Metric Select scene, where we also display the explore link in the panel menu:

https://github.com/user-attachments/assets/a60c8f83-7cb0-4431-8e71-37d7d9a67b57

Reply:
Ok, I got it working for both metric select scene and metric scene!